tests/sentry/tempest/endpoints/test_tempest_credentials.py

```diff
@@ -0,0 +1,28 @@
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.features import Feature
+
+
+class TestTempestCredentials(APITestCase):
+    endpoint = "sentry-api-0-project-tempest-credentials"
+
+    def test_create_tempest_credentials(self):
+        with Feature({"organizations:tempest-access": True}):
+            credentials = [self.create_tempest_credentials(self.project) for _ in range(5)]
+
+            other_project = self.create_project(organization=self.organization)
+            # credentials connected to other project which should not be included in the response
+            [self.create_tempest_credentials(other_project) for _ in range(5)]
```

Comment:
What is the purpose of this list comprehension that isn't persisted in any way?

Reply:
no purpose, leftover of some code cleanup. "refactored" now